core/storage/suggestion/gae_models.py

```diff
@@ -679,57 +678,24 @@ def get_reviewable_translation_suggestions_for_single_exploration(
             user_id: str. The id of the user trying to make this query.
                 Suggestions authored by this user will be excluded from
                 the results.
-
-            sort_key: str|None. The key to sort the suggestions by. If set to
-                None, the suggestions will be returned in their default order.
-                When a `sort_key` is provided, the suggestions will be sorted
-                on the client-side based on specific criteria related to their
-                presentation.
-
-            language_codes: list(str). The list of language codes.
-
+            language_code: str. The language code to get results for.
             exp_id: str. Exploration ID matching the target ID of the
                 translation suggestions.
 
         Returns:
             Tuple of (results, next_offset). Where:
                 results: list(SuggestionModel). A list of all suggestions that
```

Comment:
Explain how they're sorted, too.

Reply:
I am not sure WDYM here ? Aren't we sorting client side, then why do we need to have a explanation here ?

Reply:
Remember, this is a method all the way down the stack in the storage layer. Multiple callers may use this.

So, don't design just for your use case. The docstring needs to explain how the results are sorted when they are returned from this function, or that they are not returned in any particular order. (But in general returning in a particular order is better since it's more deterministic, so if you're in doubt, go with whatever default sorting option we currently use and sort the results using that.)

Reply:
Done